requirements.txt

```diff
@@ -2,7 +2,7 @@
 # pip install -r requirements.txt
 -e .
 cython>=0.19
-dash>=1.0.0
+dash>=1.5.1
```

Comment:
This is for the demo apps (confusing since the repo contains both the package and the demo apps)... would be better to put it here instead: https://github.com/plotly/dash-bio/blob/26df4935f5a080a7503e26b434ddac9da026d6b2/setup.py#L27